src/lib/entities/product_ingredient.py

```diff
@@ -0,0 +1,13 @@
+# entity to relate the product with the ingredient and the quantity that the product need.
+from src.utils.utils import equals
+
+
+class ProductIngredient:
+    def __init__(self):
+        self.id = None  # integer
+        self.product_id = None  # integer
```

Comment:
Shouldn't this be a product instead? Follow the same pattern all other entities have, if you are going to use _ids instead of entity object then do the same in the other existing entities